docs/reference/ACL-migration.md

```diff
@@ -0,0 +1,69 @@
+### The new ACL model
+As mentioned in our [Security Update](https://docs.lakefs.io/posts/security_update.html#whats-changing) the coming lakeFS version introduces a simplified authorization mechanism based on ACLs.
+
+### Permissions
+In the ACL model Every group will have a permission attached to it,
+lakeFS provides the following permissions:
+
+| Permission | Allows | Existing Group |
```

Comment:
Maybe we should name the "Existing Group" differently (e.g Previous Group Name) and add a comment? 
I find it a bit confusing

Reply:
Also, I think we should compare it to the permission themselves and not the groups; we can't really compare it to groups as they might have changed, while the permissions themselves weren't.